prow/config/jobs.go

```diff
@@ -145,6 +145,10 @@ type Presubmit struct {
 type Postsubmit struct {
 	JobBase
 
+	// RunIfChanged automatically run if the PR modifies a file that matches this regex.
```

Comment:
Should these changes go into JobBase itself? since both Postsubmit and Presubmit use it?

Reply:
periodic doesn't use it though

Reply:
ah.. ok

Reply:
How about a struct similar to [`Brancher`](https://github.com/kubernetes/test-infra/blob/3c763287157e7040f51d3d84fec2e1e433e9d4df/prow/config/jobs.go#L180-L191) to contain the shared logic?

Reply:
+1